### PR TITLE
Bidder/CDO can accept only one handshake

### PR DESCRIPTION
**Pull request by mjoyce91**
~~Relies on #585~~

### CODE REVIEW

File: talentmap_api/fsbid/services/bid.py [hunk 1]
```diff
@@ -98,6 +99,16 @@ def remove_bid(employeeId, cyclePositionId, jwt_token):
     return requests.delete(url, headers={'JWTAuthorization': jwt_token, 'Content-Type': 'application/json'}, verify=False)  # nosec
 
 
+def map_bids_to_disable_handshake_if_accepted(bids):
+    bidsClone = deepcopy(list(bids))
+    hasAcceptedHandshakeIds = pydash.chain(bidsClone).filter_(lambda x: pydash.get(x, 'handshake.bidder_hs_code') == 'handshake_accepted').map('id').value()
```
Comment (elizabeth-jimenez) [inline talentmap_api/fsbid/services/bid.py:104]: ⛓️ 🥇

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/bid.py:104]: Can't remember if we update the `bidder_hs_code` on revoke, but if we don't we'll want to check against that too. Ima go check that now

Comment (mjoyce91) [reply talentmap_api/fsbid/services/bid.py:104]: Good point! I guess I might as well use the normal hs_code here anyways?

Comment (elizabeth-jimenez) [reply talentmap_api/fsbid/services/bid.py:104]: yyiisss

**Review by elizabeth-jimenez — APPROVED** [review]
🍠 👍

**Pull request merged**